.github/workflows/wheels.yml

```diff
@@ -0,0 +1,119 @@
+name: Python wheels
+
+on:
+  workflow_dispatch:
+    inputs:
+      f3d_version:
+        description: 'F3D version to build'
+        required: true
+        default: 'master'
+      enable_alembic:
+        type: boolean
+        description: 'Enable Alembic plugin'
+        default: true
+      enable_assimp:
+        type: boolean
+        description: 'Enable Assimp plugin'
+        default: true
+      enable_draco:
+        type: boolean
+        description: 'Enable Draco plugin'
+        default: true
+      enable_occt:
+        type: boolean
+        description: 'Enable OpenCASCADE plugin'
+        default: true
+      enable_exr:
+        type: boolean
+        description: 'Enable OpenEXR module'
+        default: true
+
```

Comment:
Why exposing all these options? Do we need to build the wheels without certains options ?

Reply:
We do not need it to generate wheels we'll upload to pypi but I can imagine someone ask for a custom wheel someday. It does not hurt to add the options, right?

Reply:
I dont think we want untested features in CI. Not worth keeping around.